Library/ViewModels/AddNewCardViewModel.swift

```diff
@@ -42,6 +42,7 @@ public protocol AddNewCardViewModelOutputs {
   var rememberThisCardToggleViewControllerIsOn: Signal<Bool, Never> { get }
   var saveButtonIsEnabled: Signal<Bool, Never> { get }
   var setStripePublishableKey: Signal<String, Never> { get }
+  var unsupportedCardBrandError: Signal<String, Never> { get }
```

Comment:
I feel like `unsupportedCardBrandErrorText` would be more descriptive as the name for this output, since it's observed by a text label. What do you think?